internal/notify/notifications.lua

```diff
@@ -524,6 +526,25 @@ NOTIFICATIONS_BY_IDX = {
         adv_fn=curry(get_bar, get_blood, get_max_blood, "Blood", COLOR_RED),
         on_click=nil,
     },
+    {
+        name='autosave',
+        desc='Shows a reminder to save now and then.',
+        default=true,
+        dwarf_fn=function ()
+            local minsSinceSave = dfhack.persistent.getUnsavedSeconds()//60
+            if minsSinceSave < 15 then
+                return nil
+            end
+            return "Last save: ".. (dfhack.formatInt(minsSinceSave)) ..' mins ago'
+        end,
+        on_click=function ()
+            local minsSinceSave = dfhack.persistent.getUnsavedSeconds()//60
+            local message = 'It has been ' .. (dfhack.formatInt(minsSinceSave)) .. ' mins since your last save. \n\nWould you like to save now? ' ..
+            '(Note: You can still close this reminder and save manually)'
```

Comment:
```suggestion
            local message = 'It has been ' .. dfhack.formatInt(minsSinceSave) .. ' minutes since your last save. \n\nWould you like to save now? ' ..
            '(Note: You can also close this reminder and save manually)'
```